tests/OperationStore.spec.ts

```diff
@@ -64,21 +71,69 @@ async function constructAnchoredUpdateOperation (
     }]
   };
 
-  const updateOperationBuffer = await OperationGenerator.generateUpdateOperation(updatePayload, '#key1', privateKey);
+  const updateOperationBuffer = await OperationGenerator.generateUpdateOperationBuffer(updatePayload, '#key1', privateKey);
   return constructAnchoredOperation(updateOperationBuffer, transactionNumber, transactionTime, operationIndex);
 }
 
+const databaseName = 'sidetree';
+const operationCollectionName = 'operations';
+
+/**
+ * Test if a mongo service is running at the specified url
+ */
+async function isMongoServiceAvailable (serverUrl: string): Promise<boolean> {
+  try {
+    const client = await MongoClient.connect(serverUrl);
+    await client.close();
+  } catch (error) {
+    console.log('Mongoclient connect error: ' + error);
+    return false;
+  }
+  return true;
+}
+
+/**
+ * Clear a mongo collection - used to remove test state.
+ */
+async function clearMongoCollection () {
+  const client = await MongoClient.connect('mongodb://localhost:27017');
+  const db = client.db(databaseName);
+  const collections = await db.collections();
+  const collectionNames = collections.map(collection => collection.collectionName);
+
+  // If the operation collection exists, use it; else create it then use it.
+  if (collectionNames.includes(operationCollectionName)) {
+    const collection = db.collection(operationCollectionName);
+    await collection.drop();
+  }
+}
+
+async function getOperationStore (operationStoreUri: string): Promise<OperationStore> {
+  const operationStore = new MongoDbOperationStore(operationStoreUri);
+  await operationStore.initialize();
+  return operationStore;
+}
+
 describe('OperationStore', async () => {
```

Comment:
Rename this file to `MongoDbOperationStore.spec.ts` and describe as `MongoDbOperationStore`.

Reply:
Sure, I have made the change. However are we convinced that we are only going to have MongoDbOperation store going forward? The tests are quite general and will be useful for any new operation store that we might have in the future.